source/includes/portal.md

```diff
@@ -491,3 +501,660 @@ This endpoint retrieves users leaderboard in portal.
 
 <code class="request">GET /api/v1/stats/users/leaderboard</code>
 
+
+## Update the Portal
+
+
+```shell
+curl "https://community.tribe.so/api/v1/portal"
+  -X PUT
+  -H "Authorization: Bearer {access_token}"
+  --DATA '{"name": "Test Portal","domain":"test.com"}'
+```
+
+```javascript
+const tribe = require('tribe');
+
+let api = tribe.authorize('{access_token}');
+let portal = api.portal.update({ 
+  name: "Test Portal",
+  domain: "test.com"
+});
+```
+
+This endpoint updates the portal.
+
+### HTTP Request
+
+<code class="request">PUT /api/v1/portal</code>
+
+### Request Parameters
+
+Parameter | Type | Description | Required
+--------- | ----------- | ----------- | -----------
+name | `String` | The name of the portal | `Yes`
+domain | `String` | The domain of the portal | `Yes`
+subdomain | `String` | The subdomain of the portal | `No`
+announcement | `Object` | @todo | `No`
+banner | `String` | @todo | `No`
+description | `String` | Description of the portal | `No`
+email | `String` | Email of the portal | `No`
+englishName | `String` | Name of the portal in English | `No`
+favicon | `String` | Icon of the portal | `No`
+keywords | `Array<String>` | Keywords related to the portal | `No`
+links | `Object` | @todo | `No`
+logo | `String` | Logo of the portal | `No`
+longName | `String` | @todo | `No`
```

Comment:
The full name of the community. This is being used as the title of community home.